instrumentation/aws_sdk/lib/opentelemetry/instrumentation/aws_sdk/handler.rb

```diff
@@ -9,28 +9,37 @@ module Instrumentation
     module AwsSdk
       # Generates Spans for all interactions with AwsSdk
       class Handler < Seahorse::Client::Handler
-        def call(context) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+        SQS_SEND_MESSAGE = 'SQS.SendMessage'
+        SQS_SEND_MESSAGE_BATCH = 'SQS.SendMessageBatch'
+        SQS_RECEIVE_MESSAGE = 'SQS.ReceiveMessage'
+        SNS_PUBLISH = 'SNS.Publish'
+
+        def call(context) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
           return super unless context
 
-          service_name = context.client.class.api.metadata['serviceId'] || context.client.class.to_s.split('::')[1]
+          service_name = context.client.class.api.metadata['serviceId'] || context.client.class.to_s.split('::')[1] rescue context.client.class.to_s.split('::')[1] # rubocop:disable Style/RescueModifier
```

Comment:
What is this intended to do? The `rescue` clause contains the same expression as the right side of `||`, so if that raised an exception, then the `rescue` clause will as well.

Reply:
Updated service_name extraction https://github.com/open-telemetry/opentelemetry-ruby/pull/1026/commits/1a59955c5663446a47324c9193d82a7c4475337e

This change was to support `2.0.x` versions. When using `2.0.x`, `metadata` method expect an argument, what raised an exception.

Please re-review this change and approved it's safe.